packages/contract-tests/test/deploy.js

```diff
@@ -4,7 +4,7 @@ var util = require("./util");
 describe("Deployments", function () {
   var Example;
   var web3;
-  var providerOptions = { vmErrorsOnRPCResponse: false };
+  var providerOptions = { vmErrorsOnRPCResponse: false, gasLimit: 0x6691b7};
```

Comment:
ganache v2 had a default `blockGasLimit` of `0x6691b7`, which these tests rely on.